.versionrc

```diff
@@ -9,6 +9,6 @@
     }
   ],
   "scripts": {
-    "postbump": "make generate"
+    "postbump": "make generate && git add deploy/operator/kustomization.yaml"
```

Comment:
Since we already pass `-a` to standard version, do we still need `git add` here?

Reply:
yes, we do, since after `make initialize-release` with `-a` I still see the following

```diff
diff --git a/Makefile b/Makefile
index 0b33953..9780af7 100644
--- a/Makefile
+++ b/Makefile
@@ -221,12 +221,12 @@ $(STANDARD_VERSION):
 .PHONY: initiate-release
 initiate-release: $(STANDARD_VERSION)
 	git fetch git@github.com:rhobs/monitoring-stack-operator.git --tags
-	$(STANDARD_VERSION) --skip.tag # The tag will be created in the pipeline
+	$(STANDARD_VERSION) -a --skip.tag # The tag will be created in the pipeline
 
 .PHONY: initiate-release-as
 initiate-release-as: $(STANDARD_VERSION)
 	git fetch git@github.com:rhobs/monitoring-stack-operator.git --tags
-	$(STANDARD_VERSION) --skip.tag --release-as $(RELEASE_VERSION)
+	$(STANDARD_VERSION) -a --skip.tag --release-as $(RELEASE_VERSION)
 
 .PHONY: kind-cluster
 kind-cluster:
diff --git a/deploy/operator/kustomization.yaml b/deploy/operator/kustomization.yaml
index 6d66c5c..ade34d1 100644
--- a/deploy/operator/kustomization.yaml
+++ b/deploy/operator/kustomization.yaml
@@ -8,5 +8,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: monitoring-stack-operator
-  newTag: 0.0.2
+  newTag: 0.0.3
 namespace: operators
```